blocks/api/paste/utils.js

```diff
@@ -125,10 +148,10 @@ export function isPlain( HTML ) {
 	doc.body.normalize();
 
 	// If it's plain text, there should only be one node left.
-	return doc.body.childNodes.length === 1;
+	return doc.body.childNodes.length === 1 && doc.body.firstChild.nodeType === TEXT_NODE;
 }
 
-function deepFilterHelper( nodeList, filters, doc ) {
+export function deepFilterHelper( nodeList, filters, doc ) {
```

Comment:
I was worried that a helper was being exported solely for testing, but I see it's used in `inline-content-converter.js`. Is there a more descriptive name for this, to differentiate with `deepFilter` (or maybe renaming both)?

Reply:
Sure :)